### ISSUE TITLE
Menu translation

### ISSUE THREAD

**Issue by xt0ph** [jluttine/rofi-power-menu#28]
Is there a plan to translate the menu entries?
I could offer you a PR if you want.

**Comment by jluttine** [comment]
I haven't had such plans but sounds good to me. :+1: Could you describe how you'd implement it? I suppose, the script would check the language from locale settings? But this could perhaps be explicitly overridden by a CLI argument?

**Comment by xt0ph** [comment]
Yes, I check the language from locale settings. I had not thought of proposing the override by a parameter but it is easily achievable.
For the moment I'm still wondering where to put the files for the different languages. I was thinking of separating them from the script, but the installation will be less easy than now. Putting the translations in the script requires a modification for each new language.
